client/src/app/components/stock-detail-header/stock-detail-header.component.html

```diff
@@ -1,6 +1,12 @@
 <div class="company-title">
+    <span> 
```

Comment:
this should be a <div> and should have a class name to it, its bad practice to style a span / div with nothing

div = a block of elements
span = an inline element